scripts/pipeline_related.py

```diff
@@ -0,0 +1,86 @@
+from kytos.core.db import mongo_client
+from bson.decimal128 import Decimal128
+from decimal import Decimal
+
+
+client = mongo_client()
+collection = client["napps"]["flows"]
+
+# Flows from mef_eline, table groups available: epl, evpl
+query_match = {
+    "flow.cookie": {
+        "$gte": Decimal128(Decimal(12249790986447749120)),
+        "$lte": Decimal128(Decimal(12321848580485677055)),
+    },
+    "flow.match.dl_vlan": {
+        "$ne": None,
```

Comment:
> If the flow has an action related to vlan, it is an EVPL flow?

@Alopalao good point, what I'm highlighting here is that an actual EPL circuit will have a `dl_vlan` match on its nni flow entries in transport switches. The current way priority is being set works for us since it's keep it coherent in the uni ingress flow, and also, on nnis flows even though it's an EPL it'll be treated as an EVPL, which isn't wrong, initially I was with the impression that an entire EPL would always be in the same table over all switches, but this impression of mine was mistaken I think (especially since exact in_port match or exact dl_vlan + in_port could be in different tables for optimization). Now thinking about it again, looks like what you have here indeed what's expected, let's also compare with some of the initial diagrams when `of_multi_table` was specified:



![img](https://user-images.githubusercontent.com/168159/228382312-be7d0b2b-0607-46e6-8360-3fdf28180c2e.png)


Here's another example, but with a complete ring topology, check out this cookie 0xaaf36361422f3149 that was provisioned as a port-based EPL, notice the dl_vlan matches, on nnis, at is they will be categorized as `evpl` upstream, which is fine as long as it's understood that actually `evpl` table group only represents a kind of match group when an in_port + dl_vlan match is in place. I think it'd be great to exercise if with `epl` on table 0 and `evpl` on table 2 for instance it it would would work, simulating the execution on my head with a flow table miss entry matching anything from table 0 and goto table 2 I'd expect this to work, could you take the opportunity to exercise it with a ring topology to confirm that in practice it works, including confirming a ping in the data plane? It's not related to the actual change, but it's worth exercising this to confirm we're on the right path. 

```
{
    "name": "epl",
    "service_level": 7,
    "dynamic_backup_path": true,
    "uni_a": {
        "interface_id": "00:00:00:00:00:00:00:01:1"
    },
    "uni_z": {
        "interface_id": "00:00:00:00:00:00:00:03:1"
    }
}

❯ sudo ovs-ofctl -O OpenFlow13 dump-flows s1
 cookie=0xaaf36361422f3149, duration=12.518s, table=0, n_packets=0, n_bytes=0, send_flow_rem priority=20000,in_port="s1-eth4",dl_vlan=1 actions=pop_vlan,output:"s1-eth1"
 cookie=0xaaf36361422f3149, duration=12.412s, table=0, n_packets=0, n_bytes=0, send_flow_rem priority=20000,in_port="s1-eth3",dl_vlan=1 actions=pop_vlan,output:"s1-eth1"
 cookie=0xaaf36361422f3149, duration=12.521s, table=0, n_packets=0, n_bytes=0, send_flow_rem priority=10000,in_port="s1-eth1" actions=push_vlan:0x88a8,set_field:4097->vlan_vid,output:"s1

❯ sudo ovs-ofctl -O OpenFlow13 dump-flows s2
 cookie=0xaaf36361422f3149, duration=13.926s, table=0, n_packets=0, n_bytes=0, send_flow_rem priority=20000,in_port="s2-eth2",dl_vlan=1 actions=set_field:4097->vlan_vid,output:"s2-eth3"
 cookie=0xaaf36361422f3149, duration=13.923s, table=0, n_packets=0, n_bytes=0, send_flow_rem priority=20000,in_port="s2-eth3",dl_vlan=1 actions=set_field:4097->vlan_vid,output:"s2-eth2"
 
❯ sudo ovs-ofctl -O OpenFlow13 dump-flows s3
 cookie=0xaaf36361422f3149, duration=15.216s, table=0, n_packets=0, n_bytes=0, send_flow_rem priority=20000,in_port="s3-eth3",dl_vlan=1 actions=pop_vlan,output:"s3-eth1"
 cookie=0xaaf36361422f3149, duration=15.125s, table=0, n_packets=0, n_bytes=0, send_flow_rem priority=20000,in_port="s3-eth2",dl_vlan=1 actions=pop_vlan,output:"s3-eth1"
 cookie=0xaaf36361422f3149, duration=15.216s, table=0, n_packets=0, n_bytes=0, send_flow_rem priority=10000,in_port="s3-eth1" actions=push_vlan:0x88a8,set_field:4097->vlan_vid,output:"s3

```

Reply:
Excellent @Alopalao. Let's ship it as we currently have it then.